### PR TITLE
Integrate glitchtip with logrus

### PR DESCRIPTION
**Pull request by lzap**
This pull request integrates glitchtip with logrus for error tracking. It also adds logrus hook for Sentry.

To enable this feature set the following ENV vars:

```
GLITCHTIP_LOGGING=true
GLITCHTIPDSN=https://xxx@xxx.devshift.net/3
```

You will find the DSN in Glitchtip instance, project "edge-api-stage".

Then create a panic and/or log an error or fatal via logrus and see the event in Glitchtip UI.

My proposal is to merge this for edge-api pod only to see what are the errors, we might need to update some levels or change how messages are logged (typically `Errorf/Fatalf` are problem due to unique items in the error message itself). Once we get to a stable state (not many events emitted every day) we can proceed with adding other pods.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -27,16 +35,9 @@ import (
...
-
-	"github.com/Unleash/unleash-client-go/v4"
-	"github.com/getsentry/sentry-go"
-	"github.com/go-chi/chi"
-	"github.com/go-chi/chi/middleware"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/redhatinsights/platform-go-middlewares/identity"
 	"github.com/redhatinsights/platform-go-middlewares/request_id"
+	"github.com/sirupsen/logrus"
 	log "github.com/sirupsen/logrus"
```
Comment (ezr-ondrej) [inline main.go:41]: isn't this imported twice now? :thinking:

Comment (lzap) [reply main.go:41]: Oh let me fix this.

**Review by ezr-ondrej — APPROVED** [review]
Looks good to me, didn't tested it tho.
@loadtheaccumulator any thoughts? :)

**Comment by lzap** [comment]
```
[2024-04-08T12:14:11.878Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 404 - {'detail': ErrorDetail(string='Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}
[2024-04-08T12:14:11.878Z] ['verbose'] The error stack is: Error: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 404 - {'detail': ErrorDetail(string='Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}
```

Infra issue.

**Review by loadtheaccumulator — APPROVED** [review]
lgtm

**Pull request merged**